core/deployment/src/main/java/io/quarkus/deployment/builditem/FeatureBuildItem.java

```diff
@@ -48,6 +48,7 @@
     public static final String SCALA = "scala";
     public static final String SCHEDULER = "scheduler";
     public static final String SECURITY = "security";
+    public static final String SECURITY_PROPERTIES_FILE = "elytron-security-properties-file";
```

Comment:
```suggestion
    public static final String SECURITY_PROPERTIES_FILE = "security-properties-file";
```
would be better I think. We totally removed the `elytron` part from the other extension names.